packages/react-aria-components/src/Menu.tsx

```diff
@@ -398,7 +397,6 @@ function MenuItemTriggerInner<T>({item, popover, parentMenuRef, delay}: MenuItem
           trigger: 'SubmenuTrigger',
           triggerRef: ref,
           placement: 'end top',
-          UNSTABLE_portalContainer: submenuContext.popoverContainerRef.current || undefined,
```

Comment:
Oh should UNSTABLE_portalContainer really be UNSTABLE_portalContainerRef?

Reply:
Then we'd pass the ref itself? instead of taking current out?

Reply:
oh, for some reason I thought we only just added it

Reply:
Yeah it was recently added to RSP

Reply:
it is technically UNSTABLE, so i don't think changing it would count as a breaking change. however, I still think we should only do it if absolutely necessary